Makefile

```diff
@@ -586,6 +586,14 @@ test-benchmark: ## Run the benchmark tests -- Note that this can only be ran for
 	@$(GO) test -cpuprofile cpu.prof -memprofile mem.prof -bench . $(TEST_OPTIONS) $(BENCHMARK_PKG)
 	@echo "The pprof files generated are: cpu.prof and mem.prof"
 
+.PHONY: test-datastreams
+test-datastreams:
```

Comment:
Are we going to loop this in with the existing unit tests or keep it separate?

Reply:
Ahh - looks like it will be separate because we're going to invoke it from the other repository.

https://github.com/ComplianceAsCode/content/pull/11615/files#diff-c8e0190f87c4937ed23ca40b4df92b4e96cda3bfaf55755279f14e31adb41bddR49